### PR TITLE
Create README.md

### LINKED ISSUE #3: Create README.md
As a developer, I want to be able to look at the README.md for the repo contact-tracer-application to learn more about what the repo is for, the different parts of the system and how they work, and the frameworks and tools being used. 

### Acceptance Criteria: 

- [x]  There is a README.md file in the contact-tracer-application repo in the top level 

- [x] Contains a statement of purpose for repo 

- [x] Contains a section for the parts of the system

- [x] Contains a subsection for the frontend 

- [x] Contains a subsection for the backend 

- [x] Contains a subsection for the database 

- [x] Contains a section for the tools and frameworks being used 

- [x] Section for tools and frameworks is up-to-date with the current list being used

### PR DESCRIPTION
**Pull request by MCoriale**
### Issue #3 Create README.md

Created README.md in the repo and added some very basic sections to it. 

_Closes #3_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Contact Tracer Application
+<i> <ins> Last modified: October 14, 2020 </ins> </i>
```
Comment (DRSisco) [inline README.md:2]: You might not want to include this, git keeps track of this. Additionally, it's just another thing that needs to be updated manually.

Comment (DRSisco) [reply README.md:2]: If you want to fill this space below your header usually badges work great! (Google github badges) Once you know versions of current framework and languages you'll be using, this is a very good place to mention them.

**Review by DRSisco — DISMISSED** [review]
Thanks for making the README!

_and great first start to gitflow_ 🥳

File: README.md (hunk 1 above)
Comment (MCoriale) [reply README.md:2]: Awesome thanks for the feedback-- I'll remove the "Last Modified" and add some badges for what we have so far.

**Review by hanseljt77 — APPROVED** [review]

**Pull request merged**